mirrord/cli/src/error.rs

```diff
@@ -289,7 +289,7 @@ pub(crate) enum CliError {
     #[error("Feature `{0}` requires using mirrord operator")]
     #[diagnostic(help(
         "The mirrord operator is part of mirrord for Teams. \
-        You can get started with mirrord for Teams at this link: https://mirrord.dev/docs/overview/teams/"
+        You can get started with mirrord for Teams at this link: https://mirrord.dev/docs/overview/teams/?utm_source=errreqop&utm_medium=cli"
```

Comment:
Does this error not reach the IDE user? Either way, the plugins should probably also replace `utm_medium=cli`, right?

Reply:
pretty sure this specific one doesn't (at least in vscode) but i agree they should check for both